src/components/sectionsWheelPicker/index.tsx

```diff
@@ -56,7 +54,7 @@ const SectionsWheelPicker = (props: SectionsWheelPickerProps) => {
   const renderSections = () =>
     _.map(sections, (section, index) => {
       return (
-        <View height={1} width={section.style?.width || defaultWidth} key={index} testID={testID}>
+        <View height={1} width={section.style?.width} key={index} testID={testID}>
```

Comment:
Passing the `width` here is redundant cause the width should come from the wheel picker content and the style is passed there